### ISSUE TITLE
freeze model problem

### ISSUE THREAD

**Issue by buaaswf** [davidsandberg/facenet#293]
I fine tuned on the published model 0512 , It is fine
However, I load the model I saved, it occurs the following problem
`Attempting to use uninitialized value Bottleneck/BatchNorm/beta
	 [[Node: Bottleneck/BatchNorm/beta/read = Identity[T=DT_FLOAT, _class=["loc:@Bottleneck/BatchNorm/beta"], _device="/job:localhost/replica:0/task:0/cpu:0"](Bottleneck/BatchNorm/beta)]]

Caused by op u'Bottleneck/BatchNorm/beta/read', defined at:
    self.steps[index]()
 tf_batch_face_same(idcardlist, self.verify_args)
face_recognition.py", line 26, in tf_batch_face_same
    facenet.load_model(args.model_dir)
  File "/new/swf/code/facenet/src/facenet.py", line 396, in load_model
    saver = tf.train.import_meta_graph(os.path.join(model_exp, meta_file))
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/training/saver.py", line 1526, in import_meta_graph
    **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/meta_graph.py", line 502, in import_scoped_meta_graph
    producer_op_list=producer_op_list)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/importer.py", line 285, in import_graph_def
    op_def=op_def)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/ops.py", line 2240, in create_op
    original_op=self._default_original_op, op_def=op_def)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/ops.py", line 1128, in __init__
    self._traceback = _extract_stack()

FailedPreconditionError (see above for traceback): Attempting to use uninitialized value Bottleneck/BatchNorm/beta
	 [[Node: Bottleneck/BatchNorm/beta/read = Identity[T=DT_FLOAT, _class=["loc:@Bottleneck/BatchNorm/beta"], _device="/job:localhost/replica:0/task:0/cpu:0"](Bottleneck/BatchNorm/beta)]]
`

**Comment by davidsandberg** [comment]
Have you tried freezing the model using freeze_graph.py in this repo? That should be adapted to solve these kind of problems.
